node/src/effect.rs

```diff
@@ -798,6 +798,26 @@ impl<REv> EffectBuilder<REv> {
         .await
     }
 
+    /// Requests the switch block _for_ the given era ID, ie. the switch block at the era before
```

Comment:
We could use the term "key block for the given era ID" here, to avoid confusion. So the switch block of era 5 would be the key block for era 6.
(Or is that _more_ confusing? Not sure. :sweat_smile:)

Reply:
I think it could be more confusing - we don't use the term "key block" often, so I for example had to think a while whether it even makes sense ;)